src/core/raster/qgsrasterdataprovidertemporalcapabilities.h

```diff
@@ -144,7 +150,23 @@ class CORE_EXPORT QgsRasterDataProviderTemporalCapabilities : public QgsDataProv
     */
     bool isReferenceEnable() const;
 
+    /**
+     * Sets the list of available datetimes.
+     *
+     * \see dateTimes()
+     */
+    void setDateTimes( QList<QDateTime> dateTimes );
```

Comment:
I don't think this belongs here -- it's a WMS specific implementation detail and should be contained within the wms provider